vita3k/gui/src/settings_dialog.cpp

```diff
@@ -412,14 +428,27 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
 #ifdef USE_VULKAN
         static const char *LIST_BACKEND_RENDERER[] = { "OpenGL", "Vulkan" };
-        if (ImGui::Combo("Backend Renderer (Reboot for apply)", reinterpret_cast<int *>(&host.backend_renderer), LIST_BACKEND_RENDERER, IM_ARRAYSIZE(LIST_BACKEND_RENDERER)))
+        if (ImGui::Combo("Backend Renderer (Reboot to apply)", reinterpret_cast<int *>(&host.backend_renderer), LIST_BACKEND_RENDERER, IM_ARRAYSIZE(LIST_BACKEND_RENDERER)))
             host.cfg.backend_renderer = LIST_BACKEND_RENDERER[int(host.backend_renderer)];
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Select your preferred backend renderer.");
         ImGui::Spacing();
         ImGui::Separator();
         ImGui::Spacing();
 #endif
+        static const char *LIST_RES_MULTIPLIERS[] = { "1x", "2x", "3x", "4x" };
+        static int resolution_multiplier_idx = config.resolution_multiplier - 1;
+        ImGui::SetCursorPosX((ImGui::GetWindowWidth() / 2.f) - (ImGui::CalcTextSize("Upscaling").x / 2.f));
+        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Upscaling");
+        ImGui::Spacing();
+        if (!host.io.title_id.empty())
+            ImGui::BeginDisabled();
+        if (ImGui::Combo("Resolution multiplier", &resolution_multiplier_idx, LIST_RES_MULTIPLIERS, IM_ARRAYSIZE(LIST_RES_MULTIPLIERS)))
+            config.resolution_multiplier = resolution_multiplier_idx + 1;
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Enable upscaling for Vita3K. Experimental: games are not guaranteed to render properly at more than 1x");
+        if (!host.io.title_id.empty())
+            ImGui::EndDisabled();
```

Comment:
```suggestion
        if (!host.io.title_id.empty()) {
            if (ImGui::Combo("Resolution multiplier", &resolution_multiplier_idx, LIST_RES_MULTIPLIERS, IM_ARRAYSIZE(LIST_RES_MULTIPLIERS)))
                config.resolution_multiplier = resolution_multiplier_idx + 1;
            if (ImGui::IsItemHovered())
                ImGui::SetTooltip("Enable upscaling for Vita3K. Experimental: games are not guaranteed to render properly at more than 1x");
            }
```

Reply:
I think that making it disabled looks cleaner than the option literally disappearing while ingame.

Reply:
disable ? ah is no juste hide ?

Reply:
ok my bad lol